jetson/percep/perception.hpp

```diff
@@ -74,7 +74,7 @@
 
 const float inf = -std::numeric_limits<float>::infinity();
 
-const int FRAME_WRITE_INTERVAL = 3; // How many frames between writing to disk
+const int FRAME_WRITE_INTERVAL = 10; // How many frames between writing to disk
```

Comment:
Why are you changing this? This is definitely something that should be in the config file so make sure to add it to that issue so it's not forgotten.

Reply:
We plan to add all the consts in perception.hpp to the config file.

Reply:
We aren't intentionally changing it. It's always been 10, someone must have changed it to 3 in a recent commit, when I rebased I must have grabbed the 10 and not the 3